cmd/neofs-cli/internal/key/wallet.go

```diff
@@ -15,7 +15,7 @@ import (
 
 // Key-related errors.
 var (
-	ErrInvalidKey      = errors.New("provided key is incorrect")
+	ErrInvalidKey      = errors.New("provided key is incorrect, WIF and NEP2 are unsupported")
```

Comment:
It may be better to mention the expected format first.

Reply:
done